scalafmt-tests/src/test/resources/default/Select.stat

```diff
@@ -277,3 +277,78 @@ object a {
   keys.clear() // we need to remove the selected keys from the set, otherwise they remain selected
 >>>
 keys.clear() // we need to remove the selected keys from the set, otherwise they remain selected
+<<< #1334 continue chain indent after a comment
+class Foo {
+  val vv = v.aaa //
+.bbb
+.ccc()
+}
+>>>
+class Foo {
+  val vv = v.aaa //
+    .bbb.ccc()
+}
+<<< #1334 continue chain indent after a comment, a bit longer
+class Foo {
+  val vv = v.aaa //
+  .bbb //
+  .ccc //
+  .ddd
+  .eee()
+}
+>>>
+class Foo {
+  val vv = v.aaa //
+    .bbb //
+    .ccc //
+    .ddd.eee()
+}
+<<< #1334 continue chain indent after a comment with apply
+class Foo {
+  val vv = v.aaa //
+  .bbb() //
+  .ccc //
+  .ddd()
+}
+>>>
+class Foo {
+  val vv = v.aaa //
+    .bbb() //
+    .ccc //
+    .ddd()
+}
+<<< #1334 continue chain indent after a comment with apply, longer
+class Foo {
+  val vv = v.aaa //
+  .bbb() //
+  .ccc() //
+  .ddd()
+}
+>>>
+class Foo {
+  val vv = v.aaa //
+    .bbb() //
+    .ccc() //
+    .ddd()
+}
+<<< #1334 very long unbroken select chain
+class Foo {
+  val vv = v.aaaaaaaaaa.bbbbbbbbbb.cccccccccc.dddddddddd.eeeeeeeeee.ffffffffff.gggggggggg.hhhhhhhhhh.iiiiiiiiii
+}
+>>>
+class Foo {
+  val vv =
+    v.aaaaaaaaaa.bbbbbbbbbb.cccccccccc.dddddddddd.eeeeeeeeee.ffffffffff.gggggggggg.hhhhhhhhhh.iiiiiiiiii
+}
+<<< #1334 very long broken select chain
```

Comment:
This change looks unrelated to comments, the current behavior of scalafmt is to format everything in a single line.

The current behavior of Scalafmt is to never break on select nodes `a.b` unless `b` is an "applied" node like `a.b()` or `a.b[T]` or `a.b { ... }`. This is an intentional decision for better or worse, and we're going to change it then we'll need careful performance testing to ensure the search state doesn't explode in cases that formatted fine before.